docker/system_release.sh

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash
 
 #  Copyright (C) 2018-2021 LEIDOS.
```

Comment:
I think same comment for all files here to update copyright year

Reply:
yeah I missed this one. I haven't changed the License for those this PR didn't touch though